core/templates/dev/head/components/exploration_summary_tile_directive.html

```diff
@@ -175,12 +175,28 @@ <h2 class="exploration-title protractor-test-exp-summary-tile-title"><[getExplor
   .oppia-exploration-summary-tile .avatars-num-minus-one {
     color: white;
     font-family: Roboto;
-    font-size: x-small;
+    font-size: 0.55em;
     font-weight: bold;
-    left: 18px;
+    left: 12px;
     position: absolute;
-    top: 18px;
-    padding-left: 4px;
+    padding-left: 1px;
+    padding-right: 2px;
+    top: 17px;
+  }
+
+  .oppia-exploration-summary-tile .contributor-indicator {
+  border-radius: 10%;
```

Comment:
Indent each of these by 2 (the '{' at the end of the previous line creates a new indentation level). Follow the existing style in the file.